### DISCUSSION TITLE
How does the deepxde package solve a pde without training data?

### DISCUSSION THREAD

**Discussion by amiralizadeh1** [lululxvi/deepxde#1088] [Q&A]
I was looking into the examples provided, specially Allen-Cahn forward problem, I realized that the 'data' object contains only 'pde', 'geometry', and number of points on the boundary and domain. Then a model is built according to 'data' and 'net' objects. 

My question is, how does DeepXDE solve the PDE without being provided with any training data? 

Of course, there is a 'gen_testdata' function but this is only used for testing the model, not training it.

**Comment by lululxvi** [comment]
See https://doi.org/10.1137/19M1274067
